controllers/performanceprofile_controller_test.go

```diff
@@ -641,6 +641,77 @@ var _ = Describe("Controller", func() {
 				Expect(degradedCondition.Reason).To(Equal(conditionReasonMCPDegraded))
 				Expect(degradedCondition.Message).To(ContainSubstring(mcpMessage))
 			})
+
+			It("should update status when TunedProfile is degraded", func() {
+				tunedReason := "tunedReason"
+				tunedMessage := "Tuned message"
+
+				tuned := &tunedv1.Profile{
+					TypeMeta: metav1.TypeMeta{},
```

Comment:
you can omit empty structs, during the initialization it will fill it automatically, so drop
`TypeMeta: metav1.TypeMeta{}` and `Spec: tunedv1.ProfileSpec{}`